tw-entrypoints-starter/src/test/java/com/transferwise/common/entrypoints/LibraryMetricsIntTest.java

```diff
@@ -0,0 +1,17 @@
+package com.transferwise.common.entrypoints;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import com.transferwise.common.entrypoints.test.BaseIntTest;
+import org.junit.jupiter.api.Test;
+
+class LibraryMetricsIntTest extends BaseIntTest {
+
+  @Test
+  void libraryVersionIsProvided() {
+    var gauge = meterRegistry.find("tw.library.info").tags("library", "tw-entrypoints").gauge();
```

Comment:
We should do it in a centralised manner. We already have sboms created that have versions.
This looks very manual (and maybe not the right place to do those things)